package.json

```diff
@@ -11,12 +12,18 @@
   "author": {
     "name": "Hunter Tunnicliff"
   },
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=18"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": "./dist/index.js",
   "sideEffects": false,
   "files": [
     "dist",
+    "src",
```

Comment:
Does src need to be included in published files? I don't see it referenced in any export?

Reply:
Including source + source maps lets debuggers add breakpoints with the actual source code.

Reply:
Ahh - good callout